docs/integration_resources/software/deadline.md

```diff
@@ -1,5 +1,63 @@
 # Deadline
 
-Shift integration in Deadline is on the works!
+Shift integration in Deadline is in the works. The Shift plugin for Deadline is currently only available on Windows and supports execution of Shift workflows in batch mode or within Maya.
 
-If you want to go ahead and give a try to our current development for Deadline, please let us know through our [contact page](https://inbibo.co.uk/contact?_gl=1*m7h22j*_up*MQ..*_ga*MTU4MzcxNTg4OS4xNzI0MzE2MTQ2*_ga_DY05R1LZDB*MTcyNDMxNjE0Ni4xLjEuMTcyNDMxNjE1Mi4wLjAuMTg0MTg0MzEwNw..).
\ No newline at end of file
+# Shift Plugin for Deadline Renderfarm
+
+This guide explains how to set up the Shift plugin for Deadline. The setup process varies depending on the version of Python used by the Deadline installation.
+
+## Prerequisites
+
+- Access to the Deadline repository.
+- Administrative rights to create directories and modify settings in the Deadline Monitor.
+- The Shift software installed.
+
+## Step 1: Create the Plugin Folder
+
+1. Navigate to the Deadline repository at `<path_to_deadline_repository>/custom/plugins`.
+2. Create a new folder named `Shift`.
+
+## Step 2: Copy the Plugin Files
+
+1. Locate the `deadline` folder in the Shift installation directory.  
+   The path is `<path_to_the_shift_installation_folder>/deadline`.
+2. Inside the `deadline` folder, there are two directories:
+   - `python2.7`

```

Comment:
We should remove python 2.7 right?

Reply:
No, Shift uses python 3. But some companies might still have their Deadline working with python 2